internal/common/distro.go

```diff
@@ -81,3 +81,33 @@ func VersionLessThan(a, b string) bool {
 	// equal
 	return false
 }
+
+// Returns true if the version represented by the first argument is
+// semantically newer than the second.
+// Meant to be used for comparing distro versions for differences between minor
+// releases.
+// Evaluates to false if a and b are equal.
+// Assumes any missing components are 0, so 8 < 8.1.
+func VersionGreaterThan(a, b string) bool {
```

Comment:
I'm fine with adding this variant of the function to compare versions, but you could have just use `!common.VersionLessThan()` as we do in e.g.:
https://github.com/osbuild/osbuild-composer/blob/9ae8632933f8bd0eec362eaa9b37debcdc6211f5/internal/distro/rhel9/edge.go#L408 (also noticed that this example comparing `9-stream` as a version is not what the function can handle, although the outcome is definitely "correct" in this case)

https://github.com/osbuild/osbuild-composer/blob/9ae8632933f8bd0eec362eaa9b37debcdc6211f5/internal/distro/rhel9/images.go#L266

or multiple other places.

Reply:
I feel it's a bit harder to determine inclusivity for `!VersionLessThan` vs `VersionGreaterThan`.

Reply:
After more coffee I don't feel that way and it'll use `!VersionLessThan`.